js/plugins.js

```diff
@@ -28,5 +29,6 @@ module.exports = (ENV) => {
     }),
     new webpack.NamedModulesPlugin(),
     uglifyPlugin,
+    process.env.ANALYZE_BUNDLE && new BundleAnalyzerPlugin()
```

Comment:
Might be better to make this some sort of parameter for the method rather than rely on the value being pass as an environment variable. Don't have strong preference either way.